src/components/content/content.pw.tsx

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+import ContentComponent from "./components.test-pw";
+import { CHARACTERS } from "../../../cypress/support/component-helper/constants";
+import {
+  contentBody,
+  contentTitle,
+} from "../../../playwright/components/content";
+import {
+  checkAccessibility,
+  getStyle,
+} from "../../../playwright/support/helper";
+
+const testData = [CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS];
+const totalWidth = 1366;
+const alignmentData = ["left", "center", "right"];
+
+test.describe("check Content component properties", () => {
+  [
+    ["primary", "rgba(0, 0, 0, 0.9)"],
+    ["secondary", "rgba(0, 0, 0, 0.55)"],
+  ].forEach(([variant, titleColor]) => {
+    test(`should check ${variant} as variant for Content components`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<ContentComponent variant={variant} />);
+      const contentElement = contentTitle(page);
+      await expect(contentElement).toHaveCSS("color", `${titleColor}`);
+    });
+  });
+
+  testData.forEach((children) => {
+    test(`should check ${children} as children for Content component`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <ContentComponent title="Title">{children}</ContentComponent>
+      );
+
+      await expect(await contentBody(page)).toHaveText(children);
```

Comment:
**issue**: there is an unnecessary `await` within this expression

Reply:
**suggestion**: just use one await before the `expect` like this `await expect(contentBody(page)).toHaveText(children);`

Reply:
VS Code will let you know if an `await` is unnecessary by putting three dots underneath the `await` in question like so 
![Screenshot 2023-08-18 at 12 48 20](https://github.com/Sage/carbon/assets/114918852/4c3c7d47-be27-4eed-ac02-f258febd2e3b)
![Screenshot 2023-08-18 at 12 48 33](https://github.com/Sage/carbon/assets/114918852/655cfa44-908a-4479-bb51-03c33b0aa62f)